src/components/TitleClose/TitleClose.scss

```diff
@@ -1,20 +1,20 @@
 .sa-title-close {
-    .title-and-close-button {
-      display: flex;
-      justify-content: space-between;
-      align-items: center;
-    }
+  .title-and-close-button {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
 
-    .modal-title {
-      h2 {
-        @include typographyMediumM;
-        color: var(--sa-upload-modal-title-text-color);
-      }
+  .modal-title {
+    h2 {
+      @include typographyMediumM;
+      color: var(--sa-upload-modal-title-text-color);
```

Comment:
~here you can add `margin-top: 0px;`, since Modal already has a padding on it.~ EDIT: I think next comment is better

Reply:
maybe even better `margin: 0px;`, and then add some margin to the entire component, so the close button remains in the same horizontal level as the title

Reply:
> maybe even better `margin: 0px;`, and then add some margin to the entire component, so the close button remains in the same horizontal level as the title

Wow. This is good, I was struggling with this for a long time!

Reply:
I just don't understand CSS :P

Reply:
no one understands CSS :sweat_smile: